front/src/config/i18n/fr.json

```diff
@@ -791,6 +791,18 @@
         "magnet": "Capteur magnétique"
       }
     },
+    "oauth2": {
+      "buttonConnect": "Connecter maintenant",
+      "apiKeyLabel": "Saisissez ensuite votre clé client_id et secret_id ici:",
```

Comment:
client id and client secret.
In french, a space should be before `:`

Reply:
i fix it